xmtp-inbox-ios/Views/NewConversationView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  NewConversationView.swift
+//  xmtp-inbox-ios
+//
+//  Created by Elise Alix on 2/10/23.
+//
+
+import AlertToast
+import Combine
+import SwiftUI
+import XMTP
+
+struct NewConversationView: View {
+	let client: XMTP.Client
+	var onCreate: (DB.Conversation) -> Void
+	let searchTextPublisher = PassthroughSubject<String, Never>()
+
+	@Environment(\.dismiss) var dismiss
+	@FocusState var isFocused
+	@State private var searchText = ""
+	@State private var searchResults = [String]()
+	@State private var error: String?
+
+	var body: some View {
+		NavigationView {
+			ZStack {
+				Color.backgroundPrimary.edgesIgnoringSafeArea(.all)
+				List {
+					TextField("new-message-prompt", text: $searchText)
+						.focused($isFocused)
+						.onAppear {
+							self.isFocused = true
+						}
+						.onChange(of: searchText) { searchText in
+							searchTextPublisher.send(searchText)
+						}
+						.onReceive(
+							searchTextPublisher.debounce(for: .milliseconds(500), scheduler: DispatchQueue.main)
```

Comment:
I think we can just use the publisher from `searchText here. Something like:

```swift
.onReceive($searchText.debounce(for: .milliseconds(500), scheduler: DispatchQueue.main)) {
```

then we could remove the `onChange` and `searchTextPublisher`?

Reply:
I'm seeing an error trying to call `.debounce` on the binding `$searchText`. I could absolutely be doing something wrong, but will likely leave as-is for this PR!

<img width="1063" alt="Screenshot 2023-02-10 at 5 02 12 PM" src="https://user-images.githubusercontent.com/556051/218491279-b28a5ea4-4819-465a-976a-2e42c15f66b1.png">

Reply:
I'm getting an error trying to call `.debounce` on the binding `$searchText`. I could absolutely be doing something wrong but will likely leave as-is for this PR!

<img width="1175" alt="Screenshot 2023-02-13 at 9 53 00 AM" src="https://user-images.githubusercontent.com/556051/218492497-9575d18a-3158-4653-bfd2-a4ec00d3df27.png">